vagrant/ansible/roles/ctdb.setup/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Create the ctdb replicated volume
+  gluster_volume:
+    state: present
+    name: "{{ gluster_features_ctdb_volume }}"
+    bricks: '{{ ctdb_brick_location }}/ctdb'
+    cluster: "{{ gluster_cluster_hosts }}"
+    replicas: "{{ gluster_cluster_replica_count }}"
+  run_once: yes
+
+- name: Add ctdb package
+  yum:
+    name: ctdb
+    state: present
+
+- name: Add ctdb rules to firewalld
+  firewalld:
+    service: ctdb
+    permanent: yes
+    state: present
+    state: enabled
+
+- name: Add recovery lock to ctdb.conf
+  lineinfile:
+    dest: /etc/ctdb/ctdb.conf
+    line: "\trecovery lock = /gluster/lock/recovery.lock"
+    regexp: "recovery lock"
+    insertafter: '\[cluster\]'
+
+- name: Add netbios name - The two ctdb nodes should have the same netbios name
+  lineinfile:
+    dest: /etc/samba/smb.conf
+    line: "netbios name = {{ ctdb_netbios_name }}"
+    regexp: "netbios name"
+    insertafter: '\[global\]'
+
+
+# Check bz 1656777
+- name: Create missing directories for ctdb package
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: root
+  with_items:
+    - /etc/ctdb/events/legacy
```

Comment:
Why do these need to be created? Should come from the RPM, no?

Reply:
centos7 ctdb packaging issues. Currently added as a block. Will remove once the problems with the package have been resolved.